src/main/webapp/util/Utils.js

```diff
@@ -45,3 +48,55 @@ export const scrollToTop = () => {
 export const handleRedirectToProject = (serverURL, projectKey) => {
   return [serverURL, "project", "main?projectKey=" + projectKey + "&tab=review"].join("/");
 }
+
+// columns headers should be included in the first row in data array.
+// Eg of data : [['header1', 'header2', 'header3'], ['row1value1', 'row1value2', 'row1value3'], ['row2value1', 'row2value2', 'row2value3']]
```

Comment:
nit: Would be more legible if we make this multiline
```suggestion
// Eg of data : [['header1', 'header2', 'header3'], 
//               ['row1value1', 'row1value2', 'row1value3'], 
//               ['row2value1', 'row2value2', 'row2value3']]
```

Reply:
Yeah, it looks more legible now.